integration-test/background/click-to-load-integration-tests.js

```diff
@@ -3,6 +3,7 @@ const harness = require('../helpers/harness')
 const testSite = 'https://privacy-test-pages.glitch.me/privacy-protections/click-to-load/'
 let browser
 let bgPage
+/** DISABLED due to perma failing
 
 describe('Test Click To Load', () => {
```

Comment:
```suggestion
xdescribe('Test Click To Load', () => {
```
See https://jasmine.github.io/api/2.7/global.html#xdescribe

Reply:
Ah thanks TIL